src/index.html

```diff
@@ -9,10 +9,61 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="styles/style.scss"
```

Comment:
It appears you are trying to link a SCSS file directly in your HTML. Web browsers do not understand SCSS files directly. You should compile your SCSS to CSS and link the resulting CSS file here.